tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -261,21 +262,29 @@ def parse_args():
     return parser.parse_args()
 
 
-def convert_onnx_models_to_ort():
-    args = parse_args()
-
-    output_dir = None
-    if args.output_dir is not None:
-        if not args.output_dir.is_dir():
-            args.output_dir.mkdir(parents=True)
-        output_dir = args.output_dir.resolve(strict=True)
+def convert_onnx_models_to_ort(
+    model_path_or_dir: pathlib.Path,
+    output_dir: pathlib.Path | None = None,
+    optimizations: list[str] | None = None,
```

Comment:
can it be `optimization_styles: list[OptimizationStyle]`?

we could update `parse_args()` to fix up the type, e.g.:
```
args = parser.parse_args()
args.optimization_style = [OptimizationStyle[style_str] for style_str in args.optimization_style]
return args
```